playbooks/yarn.yml

```
@@ -18,6 +18,7 @@
 - import_playbook: yarn_ranger_config.yml
 
 - import_playbook: yarn_ranger_init.yml
+- import_playbook: yarn_jmx-exporter_config.yml
```

```suggestion
- import_playbook: yarn_ranger_config.yml
- import_playbook: yarn_jmx-exporter_config.yml

- import_playbook: yarn_ranger_init.yml
```

A config task should stay in a config block if possible

---

done.